abfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>remove the mention of the ligand network? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>